platform/server/src/main/webapp/static/js/bootstrap-page.js

```diff
@@ -60,6 +63,69 @@ function verifyDbConnection() {
     });
 }
 
+
+function verifyAmqConnection() {
```

Comment:
Looks awful familiar this function. We should reuse the code rather then copy paste it